docs/IHO/b12_v3_2_0-BETA_example.json

```
@@ -0,0 +1,133 @@
+{
+  "type": "FeatureCollection",
+  "crs": {
+    "type": "name",
+    "properties": {
+      "name": "EPSG:4326"
+    }
+  },
+  "properties": {
+    "trustedNodePlatform": {
+      "providerOrganizationName": "Sea-ID",
+      "providerEmail": "support@sea-id.org",
+      "uniqueVesselID": "SEAID-e8c469f8-df38-11e5-b86d-9a79f06e9478",
+      "type": "Private vessel",
+      "name": "White Rose of Drachs",
+      "IDType": "MMSI",
+      "IDNumber": "369958000"
+    },
+    "observationCollection": {
+      "trustedNode": {
+        "convention": "GeoJSON CSB 3.2",
+        "dataLicense": "CC0 1.0",
+        "providerLogger": "Rose Point ECS",
+        "providerLoggerVersion": "1.0",
+        "navigationCRS": "EPSG:4326",
+        "verticalReferenceOfDepth": "Transducer",
+        "vesselPositionReferencePoint": "GNSS"
+      },
+      "platform": {
+        "length": 65,
+        "sensorDescriptions": [
+          {
+            "type": "Sounder",
+            "make": "Garmin",
+            "model": "GT-50",
+            "position": [
+              4.2,
+              0.0,
+              5.4
+            ],
+            "draft": 1.4,
+            "draftUncert": 0.2,
+            "frequency": 200000
+          },
+          {
+            "type": "GNSS",
+            "make": "Litton Marine Systems",
+            "model": "LMX420"
+          }
+        ],
+        "soundSpeedDocumented": true,
+        "positionOffsetsDocumented": true,
+        "dataProcessed": true,
+        "contributorComments": "On 2022-03-08, at 20:30 UTC, the echo sounder lost bottom tracking after the vessel crossed another vessel's wake."
+	    },
+      "processing": [
+        {
+          "type": "soundSpeed",
+          "timestamp": "2023-02-14T01:00:00.0000Z",
+          "method": "Interpolated RMC Messages",
```

Not an issue for the code, but this wouldn't be a method to generate sound speed corrections.
```suggestion
          "method": "WOA18 Profile",
```

---

This has been resolved in a049f3ff57ca6c5cd27233d9fbdf264c2a8033a7.